### PR TITLE
Updating CHANGELOG.md and branding from release-1.3.x branches.

### PR DESCRIPTION
**Pull request by sacherjj**
Pulling any CHANGELOG entries from 1.3.x back into dev.
Updating branding to correct lingering CasperLabs references and old repos.

### CODE REVIEW

File: resources/production/chainspec.toml [hunk 1]
```diff
@@ -148,7 +148,7 @@ call_contract = { cost = 4_500, arguments = [0, 0, 0, 0, 0, 420, 0] }
 call_versioned_contract = { cost = 200, arguments = [0, 0, 0, 0, 0, 0, 0, 0, 0] }
 create_contract_package_at_hash = { cost = 200, arguments = [0, 0] }
 create_contract_user_group = { cost = 200, arguments = [0, 0, 0, 0, 0, 0, 0, 0] }
-create_purse = { cost = 2_500_000_000, arguments = [0, 0] }
+create_purse = { cost = 170_000, arguments = [0, 0] }
```
Comment (Fraser999) [inline resources/production/chainspec.toml]: I think this change should be reverted as it was changed to `2_500_000_000` in #1326 recently.

Same point for `[system_costs.mint_costs.mint]` and `[system_costs.mint_costs.create]` below.

Changes to `[system_costs.auction_costs.add_bid]` and `[system_costs.auction_costs.delegate]` appear to be correct here though, having been changed to these values in #1834.

Comment (sacherjj) [reply resources/production/chainspec.toml]: Corrected.

**Review by Fraser999 — APPROVED** [review]

**Comment by sacherjj** [comment]
bors merge

**Comment by sacherjj** [comment]
bors merge

**Pull request merged**